### ISSUE TITLE
Remove prompt to choose session file name

### ISSUE THREAD

**Issue by codefromthecrypt** [block-open-source/goose#89]
### Summary

I'd like a way to end my session without a follow-up question about saving the session.

### Background

Right now, exiting `goose session start` is a two step process, as first you need to break the loop (e.g. by "exit") then deal with the following question:
```
Enter a name to save this session under. A name will be generated for you if empty:                                                                                                                        
```

If you don't want to save it, you ctrl-c, if you just want to get rid of this message, you hit enter, otherwise you choose a name.

This name is used in resume, so if you choose the session name "apples" you would do `goose session resume apples`

### Preferred Implementation

I would prefer to have no prompt, and use a generated name by default. If you pass a session name, then have that used.

Concretely, if you want to use `goose session resume apples`, then start via `goose session start apples`.

### Rationale

This is very little code to make parity between the session start and resume commands. This seems intuitive as even if we did think the user should know about sessions, they still need to know how to pass the same arg to resume the session. Since that's the case, I think just use the same way.

**Comment by codefromthecrypt** [comment]
ps I'm happy to raise a PR and test it

**Comment by codefromthecrypt** [comment]
fixed by #94
